.claude/CLAUDE.md

```
@@ -1,19 +1,33 @@
+michi --version
+ls ~/.claude/plugins/marketplaces/sk8metal-michi/plugins/michi/
+```
+
+**インストール方法:**
+```
+/plugin marketplace add sk8metalme/michi
+/plugin install michi@sk8metalme
+npm install -g @sk8metal/michi-cli
+```
+
+詳細: https://github.com/sk8metalme/michi
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)